src/frontend/pages/waiting-room.tsx

```diff
@@ -1,10 +1,71 @@
 import { useRouter } from 'next/dist/client/router';
-import { ReactElement } from 'react';
+import { ReactElement, useState } from 'react';
+import { DEV_URL } from '../api-client';
+import Timer from '../components/timer';
+import { useEventSource } from '../hooks/useEventSource';
+import styles from '../styles/WaitingRoom.module.scss';
 
+const MAX_PLAYERS = 4;
 export default function WaitingRoom(): ReactElement {
   const router = useRouter();
 
   const { playerId } = router.query;
```

Comment:
I should have mentioned this is in the previous PR, but would it be possible to query the userID as well? (like the one they input) 

I think it would be confusing if they signed-in as user '1234' and then we call them player 5 all the time.

Reply:
We can do that, we would just have to change the waiting room controller to find the player based on their playerId, but I don't think we currently demand uniqueness by userId, so that might mess with some of the logic (all the players i've created have the same userId)

Reply:
I can make that refactor in a different pr?

Reply:
I think we should query based on our generated playerIds, but maybe also send back the userId for display purposes.

Reply:
i kind of feel like having both the playerid and user id in the query string for the page is redundant. Besides I'm not sure how many users look at the query string in the header.

Reply:
ok fair enough, I think that's fine then